packages/task/src/browser/task-service.ts

```diff
@@ -129,14 +138,16 @@ export class TaskService implements TaskConfigurationClient {
     protected init(): void {
         // notify user that task has started
         this.taskWatcher.onTaskCreated((event: TaskInfo) => {
-            if (this.isEventForThisClient(event.ctx)) {
-                const task = event.config;
-                let taskIdentifier = event.taskId.toString();
-                if (task) {
-                    taskIdentifier = !!this.taskDefinitionRegistry.getDefinition(task) ? `${task._source}: ${task.label}` : `${task.type}: ${task.label}`;
-                }
-                this.messageService.info(`Task ${taskIdentifier} has been started`);
+            if (!this.isEventForThisClient(event.ctx)) {
+                return;
             }
+            this.runningTasks.set(event.taskId, { exitCode: new Deferred<number | undefined>(), terminateSignal: new Deferred<string | undefined>() });
+            const task = event.config;
+            let taskIdentifier = event.taskId.toString();
+            if (task) {
+                taskIdentifier = !!this.taskDefinitionRegistry.getDefinition(task) ? `${task._source}: ${task.label}` : `${task.type}: ${task.label}`;
```

Comment:
The PR changes approach of displaying labels for `TaskRunQuickOpenItem`s.
For example, task 
```
   {
      "type": "shell",
      "label": "taskStart",
      "command": "sleep 10 && echo test start task"
    }
```
is displayed as `TaskRunQuickOpenItem` with label `taskStart`
But for notifications old approach is used: `Task shell:taskStart has been started`
Maybe it makes sense to align this part as well.  
WDYT?

Reply:
Fixed